observatorium/grafana-data-sources/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - opf-test-loki-source.yaml
```

Comment:
Should we remove references to `test` from all the files?

Reply:
This is a test datasource that can be used as an example to create others

Reply:
This was copied from https://github.com/operate-first/apps/blob/master/odh/base/monitoring/grafana.yaml

Reply:
Also this is for the org-id `'opf-test'`